pkgs/development/python-modules/osqp/default.nix

```diff
@@ -23,16 +22,15 @@ buildPythonPackage rec {
   dontUseCmakeConfigure = true;
 
   propagatedBuildInputs = [
-    numpy
     future
+    numpy
```

Comment:
Why add these new dependencies? Are they required for the package?

Reply:
Re numpy/scipy: They seem to be used in the code, it's just not listed in requirements.txt
which I used when originally packaging. It seems to still build/test fine, probably due to them being added to PYTHONPATH by some upstream package.

Reply:
re scipy: it'd originally been a test dependency, but I wanted to move it to a regular dependency b/c I found the above code in it.